tldw_Server_API/tests/DB_Management/test_content_backend_cache.py

```
@@ -169,3 +269,246 @@ def fake_create(cfg):
+
+    def stamp_abtest_store(self) -> None:
+        stamp = self._backend_target_key(self._backend) or "<none>"
+        self._abtest_store = stamp
+        stamps.append(stamp)
+
+    monkeypatch.setattr(evaluations_db.EvaluationsDatabase, "_init_abtest_store", stamp_abtest_store)
+
+    db = evaluations_db.EvaluationsDatabase(db_path=None)
+
+    holder["backend"] = new_backend
+    db.backend
```

Ruff (0.15.10)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)